driver/csiplugin/controllerserver.go

```diff
@@ -921,9 +934,9 @@ func (cs *ScaleControllerServer) validateSrcVolumeID(vID *scaleVolId, scVol *sca
 	}
 
 	// This is kind of snapshot restore
```

Comment:
what does this comment mean here?

Reply:
In Snapshot restore, source is snapshot and destination is PVC. In Volume cloning, source is PVC and destination is PVC. So as copy to PVC happens, it is mentioned as kind of snapshot restore.